src/config.rs

```diff
@@ -441,6 +441,15 @@ impl Config {
             false
         }
     }
+
+    pub fn get_allow_failure(&self, matches: &ArgMatches) -> bool {
+        matches.is_present("allow_failure")
+            || if let Ok(var) = env::var("SENTRY_ALLOW_FAILURE") {
```

Comment:
Nice, it's much easier to document the `SENTRY_ALLOW_FAILURE` env. var. for plugins.